vital/types/object_track_set.h

```diff
@@ -48,6 +48,8 @@
 
 #include <vital/range/transform.h>
 
+#include <vital/types/point.h>
```

Comment:
FYI, `vital/types` s.b. before `vital/range`, and probably before (or maybe with) `vital/*.h`. (`vital/range` is "below" pretty much everything else; they are STL-like headers and have no external dependencies.)

Reply:
Ok, I took a shot.
I was just going by the alphabet